components/SdkItem.tsx

```diff
@@ -0,0 +1,211 @@
+import merge from 'lodash/merge';
+import Link from 'next/link';
+import React from 'react';
+
+const itemId = {
+    1: 'Quickstart',
+    2: 'Sandbox',
+    3: 'Sample app',
+    4: 'API reference',
+    5: 'Release notes',
+    6: 'Webhook',
+    7: 'Domain and port whitelisting'
+};
+
+const iconStyle = { height: '20px', color: '$textMedEmp' };
+
+const itemList = {
+    1: <BookIcon style={iconStyle} />,
+    2: <CodeIcon style={iconStyle} />,
+    3: <GithubIcon style={iconStyle} />,
+    4: <RestApiIcon style={iconStyle} />,
+    5: <RocketIcon style={iconStyle} />,
+    6: <WebhookIcon style={iconStyle} />,
+    7: <ListCheckIcon style={iconStyle} />
+};
+
+const sdkItems = {
+    javascript: [
+        { id: 1, link: '/javascript/v2/guides/javascript-quickstart' },
+        { id: 2, link: 'https://codesandbox.io/s/hms-js-quickstart-5rmes' },
+        { id: 4, link: '/api-reference/javascript/v2/home/content' },
+        { id: 5, link: '/javascript/v2/release-notes/release-notes' }
+    ],
+    react: [
+        { id: 1, link: '/javascript/v2/guides/javascript-quickstart' },
+        { id: 3, link: 'https://github.com/100mslive/100ms-web' },
+        { id: 4, link: '/api-reference/javascript/v2/home/content' },
+        { id: 5, link: '/javascript/v2/release-notes/release-notes' }
+    ],
+    serverSide: [
+        // { id: 1, link: '/server-side/v2/guides/quickstart' },
+        { id: 4, link: '/server-side/v2/foundation/templates-and-roles#api-reference' },
+        { id: 6, link: '/server-side/v2/foundation/webhook' },
+        { id: 7, link: '/server-side/v2/foundation/firewall-and-ports' }
+    ],
+    android: [
+        { id: 1, link: '/android/v2/guides/Quickstart' },
+        { id: 3, link: 'https://github.com/100mslive/100ms-android' },
+        { id: 4, link: '/api-reference/android/v2/index.html' },
+        { id: 5, link: '/android/v2/release-notes/Release-Notes' }
+    ],
+    ios: [
+        { id: 1, link: '/ios/v2/guides/Quickstart' },
+        { id: 3, link: 'https://github.com/100mslive/100ms-ios-sdk' },
+        { id: 4, link: '/api-reference/ios/v2/home/content' },
+        { id: 5, link: '/ios/v2/release-notes/Release-Notes' }
+    ],
+    flutter: [
+        { id: 1, link: '/flutter/v2/guides/quickstart' },
+        { id: 3, link: 'https://github.com/100mslive/100ms-flutter/tree/main/example' },
+        {
+            id: 4,
+            link: 'https://pub.dev/documentation/hmssdk_flutter/latest/hmssdk_flutter/hmssdk_flutter-library.html'
+        },
+        { id: 5, link: '/flutter/v2/release-notes/release-notes' }
+    ],
+    reactNative: [
+        { id: 1, link: '/react-native/v2/guides/quickstart' },
+        { id: 3, link: 'https://github.com/100mslive/react-native-hms' },
+        { id: 4, link: '/api-reference/react-native/v2/modules.html' },
+        { id: 5, link: '/react-native/v2/release-notes/release-notes' }
+    ]
+};
+
+interface Props {
+    logo?: React.SVGProps<SVGSVGElement>;
+    sdk?: any;
```

Comment:
should be string

Reply:
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/64203150/189726095-be7a1989-0dd1-4195-b22e-181a5ed78119.png">